config/packages/security.yaml

```diff
@@ -34,5 +34,8 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        # - { path: ^/admin, roles: ROLE_ADMIN }
-        # - { path: ^/profile, roles: ROLE_USER }
+        -  {  path:  ^/login ,  roles:  IS_AUTHENTICATED_ANONYMOUSLY  } 
+        -  {  path:  ^/booking/ ,  roles:  ROLE_ADMIN  }
+        -  {  path:  ^/agent/ ,  roles:  ROLE_ADMIN  }
+        -  {  path:  ^/dashboard/ ,  roles:  ROLE_ADMIN  }
+        -  {  path:  ^/ ,  roles:  ROLE_ADMIN  }
```

Comment:
You should clarify this because your path are not exclusive

Reply:
i didn't understand, what do you mean when you say "path are not exclusive" ?